ibis/backends/mssql/__init__.py

```diff
@@ -84,6 +85,41 @@ def list_databases(self, like: str | None = None) -> list[str]:
     def current_schema(self) -> str:
         return self._scalar_query(sa.select(sa.func.schema_name()))
 
+    def list_tables(
+        self,
+        like: str | None = None,
+        database: str | None = None,
+        schema: str | None = None,
+    ) -> list[str]:
+        tablequery = sg.select("name").from_(
+            sg.exp.Table(this="tables", catalog=database, db="sys")

```

Comment:
We should use `sg.table` here. `sg.exp.Table` doesn't automatically handle quoting:

```
In [4]: import sqlglot as sg

In [5]: sg.exp.Table(this="a b c", catalog="d e f").sql("duckdb")
Out[5]: 'd e f.a b c'

In [6]: sg.table("a b c", catalog="d e f").sql("duckdb")
Out[6]: '"d e f"."a b c"'
```

Generally wherever you don't need to handle parsing, prefer `sg.table` over `sg.exp.Table`